### PR TITLE
Luk10114/1275 read symbols mobile style file

### PR DESCRIPTION
**Pull request by LDuncAndroid**
https://devtopia.esri.com/runtime/common-samples/issues/1275

### CODE REVIEW

File: java/read-symbols-mobile-style-file/src/main/java/com/esri/arcgisruntime/sample/readsymbolsmobilestylefile/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+  private GraphicsOverlay mGraphicsOverlay;
+  private RecyclerView mEyesRecyclerView;
+  private RecyclerView mMouthRecyclerView;
+  private RecyclerView mHatRecyclerView;
+  private ImageView mPreviewView;
+  private SymbolAdapter mEyesAdapter;
+  private SymbolAdapter mMouthAdapter;
+  private SymbolAdapter mHatAdapter;
+
+  private SymbolStyleSearchResultObservable symbolStyleSearchResultObservable = new SymbolStyleSearchResultObservable();
```
Comment (TADraeseke) [inline java/read-symbols-mobile-style-file/src/main/java/com/esri/arcgisruntime/sample/readsymbolsmobilestylefile/MainActivity.java]: hungarian casing if making this a field

File: java/read-symbols-mobile-style-file/src/main/java/com/esri/arcgisruntime/sample/readsymbolsmobilestylefile/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+  private class SymbolStyleSearchResultObservable extends Observable {
+
+    private List<SymbolStyleSearchResult> mSymbols;
+
+    List<SymbolStyleSearchResult> getSymbols() {
+      return mSymbols;
+    }
+
+    void setSymbols(List<SymbolStyleSearchResult> symbols) {
+      if (symbols.size() > 0) {
```
Comment (TADraeseke) [inline java/read-symbols-mobile-style-file/src/main/java/com/esri/arcgisruntime/sample/readsymbolsmobilestylefile/MainActivity.java]: !symbols.isEmpty()

File: java/read-symbols-mobile-style-file/src/main/java/com/esri/arcgisruntime/sample/readsymbolsmobilestylefile/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,431 @@
...
+    @Override public void onBindViewHolder(@NonNull ViewHolder viewHolder, int i) {
+      viewHolder.bind(symbols.get(i), mOnSymbolPreviewTapListener);
+    }
+
+    @Override public int getItemCount() {
+      return symbols.size();
+    }
+
+    void addSymbol(SymbolStyleSearchResult symbol) {
+      this.symbols.add(symbol);
```
Comment (TADraeseke) [inline java/read-symbols-mobile-style-file/src/main/java/com/esri/arcgisruntime/sample/readsymbolsmobilestylefile/MainActivity.java]: drop this and use hungarian casing

File: java/read-symbols-mobile-style-file/src/main/java/com/esri/arcgisruntime/sample/readsymbolsmobilestylefile/MainActivity.java [hunk 4]
```diff
@@ -0,0 +1,431 @@
...
+            mMouthAdapter.addSymbol(symbol);
+            break;
+          case "hat":
+            mHatAdapter.addSymbol(symbol);
+            break;
+          case "face":
+            mFaceSymbolKey = symbol.getKey();
+            break;
+        }
+        Log.d(TAG, symbol.getCategory());
```
Comment (TADraeseke) [inline java/read-symbols-mobile-style-file/src/main/java/com/esri/arcgisruntime/sample/readsymbolsmobilestylefile/MainActivity.java]: This is probably quite useful for now, but lets make sure we remove it if we don't need it for the final sample

File: java/read-symbols-mobile-style-file/src/main/java/com/esri/arcgisruntime/sample/readsymbolsmobilestylefile/MainActivity.java [hunk 5]
```diff
@@ -0,0 +1,431 @@
...
+        ListenableFuture<Bitmap> bitmapFuture = symbol.getSymbol().createSwatchAsync(itemView.getContext(),
+            Color.TRANSPARENT);
+        try {
+          // wait for the Future to complete and get the result
+          Bitmap bitmap = bitmapFuture.get();
+          mImageView.setImageBitmap(bitmap);
+        } catch (InterruptedException | ExecutionException e) {
+          logErrorToUser(itemView.getContext(), getString(R.string.error_loading_symbol_bitmap_failed, e.getMessage()));
+        }
+        itemView.setOnClickListener(v -> {
```
Comment (TADraeseke) [inline java/read-symbols-mobile-style-file/src/main/java/com/esri/arcgisruntime/sample/readsymbolsmobilestylefile/MainActivity.java]: can be moved to a 1 line lambda

**Comment by LDuncAndroid** [comment]
@TADraeseke updated and ready for re-review. thanks!

**Comment by LDuncAndroid** [comment]
@TADraeseke assigning to you as I believe @hardikc07 has some issues with the sample.

Hardik: am seeing an exception of file not found

**Comment by LDuncAndroid** [comment]
Closed due to dupe https://github.com/Esri/arcgis-runtime-samples-android/pull/617

**Pull request closed without merging**